src/coreclr/vm/loaderallocator.cpp

```diff
@@ -1734,11 +1734,7 @@ void AssemblyLoaderAllocator::Init(AppDomain* pAppDomain)
     LoaderAllocator::Init((BaseDomain *)pAppDomain);
     if (IsCollectible())
     {
-        // TODO: the ShuffleThunkCache should really be using the m_pStubHeap, however the unloadability support
-        // doesn't track the stubs or the related delegate classes and so we get crashes when a stub is used after
-        // the AssemblyLoaderAllocator is gone (the stub memory is unmapped).
-        // https://github.com/dotnet/runtime/issues/55697 tracks this issue.
```

Comment:
was there any recent change which made tracking of stubs during unload? Just want to check whether it might still cause intermittent issues?

Reply:
I am not aware of a specific change in this area, however, it has been a couple of years since I have experienced the problem when developing the unloadability. And even at that time, I was unable to tell how that could happen.

Reply:
It is possible that there is still a problem hidden somewhere and happening rarely. I will try to run some extra gc stress tests when running the tests in an unloadable context.